PocketCastsTests/Tests/Player/Transcripts/ComposeFilterTests.swift

```diff
@@ -0,0 +1,53 @@
+import XCTest
+
+@testable import podcasts
+
+final class ComposeFilterTests: XCTestCase {
+
+    func testNormalString() throws {
+        let transcript = """
+        This content should not change
+        """
+
+        let filtered = ComposeFilter.transcriptFilter.filter(transcript)
+
+        XCTAssertEqual(filtered, "This content should not change ")
+    }
+
+    func testSpeakerX() throws {
+        let transcript = """
+        Speaker 1: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus imperdiet condimentum ligula quis fringilla. Donec eu ultricies enim. Aenean eu risus leo. Nulla facilisi.
+        """
+
+        let filtered = ComposeFilter.transcriptFilter.filter(transcript)
+
+        let expected = """
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+        Phasellus imperdiet condimentum ligula quis fringilla.
+        Donec eu ultricies enim.
+        Aenean eu risus leo.
+        Nulla facilisi.
```

Comment:
I feel like breaking sentences based on period might make it harder to read. This is an example, I feel like the original is way easier. 😅

I do understand that a good balance is hard to achieve and on real podcasts it seems ok, so it might be just that this example is really not a good one.

Reply:
I researched to try to find good metrics, we could say for example to do full stop every two sentences or x amount of words but it's heavily influenced by the context of the writing itself.